.github/workflows/macos.yml

```diff
@@ -61,174 +76,55 @@ jobs:
-          python --version
-          scons --version
-
-      - name: Compilation
+      - name: Compilation (template)
         working-directory: ./scripts
+        if: ${{ matrix.template }}
         env:
           SCONS_CACHE: ${{github.workspace}}/.scons_cache/
         run: |
-          scons platform=macos tools=yes target=editor extra_suffix=luaAPI luaapi_luaver=jit
-      - name: Prepare artifact
-        run: |
-          strip scripts/godot/bin/godot.*
-          chmod +x scripts/godot/bin/godot.*
-      - uses: actions/upload-artifact@v2
-        with:
-          name: ${{github.job}}
-          path: scripts/godot/bin
-
-  macos-template:
-    runs-on: "macos-latest"
-    name: MacOS Template (target=template_release, tools=no)
-
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          submodules: recursive
-
-      - name: Setup Vulkan SDK
-        run: |
-          sh scripts/install_vulkan_sdk_macos.sh
-
-      - name: Load .scons_cache directory
-        id: macos-template-cache
-        uses: actions/cache@v2
-        with:
-          path: ${{github.workspace}}/.scons_cache/
-          key: ${{github.job}}-${{env.BASE_BRANCH}}-${{github.ref}}-${{github.sha}}
-          restore-keys: |
-            ${{github.job}}-${{env.BASE_BRANCH}}-${{github.ref}}-${{github.sha}}
-            ${{github.job}}-${{env.BASE_BRANCH}}-${{github.ref}}
-            ${{github.job}}-${{env.BASE_BRANCH}}
-
-      - name: Set up Python 3.x
-        uses: actions/setup-python@v2
-        with:
-          python-version: "3.x"
-          architecture: "x64"
-
-      - name: Configuring Python packages
-        run: |
-          python -c "import sys; print(sys.version)"
-          python -m pip install scons
-          python --version
-          scons --version
+          scons platform=macos arch=x86_64 target=template_debug ${{ env.SCONSFLAGS }}
+          scons platform=macos arch=arm64 target=template_debug ${{ env.SCONSFLAGS }}
+          scons platform=macos arch=x86_64 target=template_release ${{ env.SCONSFLAGS }}
+          scons platform=macos arch=arm64 target=template_release ${{ env.SCONSFLAGS }}
 
-      - name: Compilation
+      - name: Make universal binaries and bundle (editor)
         working-directory: ./scripts
-        env:
-          SCONS_CACHE: ${{github.workspace}}/.scons_cache/
-        run: |
-          scons platform=macos target=template_release tools=no extra_suffix=luaAPI
-      - name: Prepare artifact
+        if: ${{ matrix.editor }}
         run: |
-          strip scripts/godot/bin/godot.*
-          chmod +x scripts/godot/bin/godot.*
-      - uses: actions/upload-artifact@v2
-        with:
-          name: ${{github.job}}
-          path: scripts/godot/bin
-
-  macos-template-debug:
-    runs-on: "macos-latest"
-    name: MacOS Template Debug (target=template_debug, tools=no)
+          lipo -create bin/godot.macos.editor.x86_64 bin/godot.macos.editor.arm64 -output bin/godot.macos.editor.universal
```

Comment:
`lipo -create bin/godot.macos.editor.x86_64 bin/godot.macos.editor.arm64 -output bin/godot.macos.editor.universal`
Should be
`lipo -create scripts/bin/godot.macos.editor.x86_64 scripts/bin/godot.macos.editor.arm64 -output bin/godot.macos.editor.universal`